client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:16-alpine 
+WORKDIR /app
+COPY . .
+RUN npm ci 
```

Comment:
The dependancies can be installed separately for the build and run stage using multi staged builds to reduce the final image size